demos-europe/demosplan-core
# feat: fixes some Addon autoloading issues

This is fixing some issues regarding addon installation and activation, including autoloading issues and rebuilding the Symfony container.

There most likely will be more changes and fixes ahead (as some parts are still unfinished or at least untested), but it seemed reasonable enough to make this a PR.

### How to review/test
Check it out and install an installable Addon. It should install it correctly. If you try to activate it, there should be errors.
